src/Pages/Auth/Login/index.js

```diff
@@ -1,11 +1,35 @@
-import React from "react"
+import React, { useEffect } from "react"
+import { Link } from "react-router-dom"
 
 Login.propTypes = {}
 
-function Login() {
+function Login(props) {
+  const loggedStatous = props.userIsLogged
```

Comment:
No need to copy the props to a local variable here. You can reuse props.userIsLogged or destructure the props like
`const { userIsLogged } = props`

Reply:
I dont quite understand this. How can 'outer' Login (the component) have no properties and on the same time the 'inner' login (the function) have properties? Where does it get them from?